arm-resources/links/2016-09-01/swagger/links.json

```diff
@@ -0,0 +1,300 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ManagementLinkClient",
+    "version": "2016-09-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{linkId}": {
+      "delete": {
+        "tags": [
+          "ResourceLinks"
+        ],
+        "operationId": "ResourceLinks_DeleteById",
```

Comment:
Normally, I'd keep the ById part in operation name, if we had another path that lets you delete a link. Like, /scope/Microsoft.Resources/links/{linkName}. I think this is redundant, so I'm fine changing the operation name under this path to Delete, Get and CreateOrUpdate, instead of DeleteById, GetById and CreateOrUpdateById